purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -30,6 +32,21 @@ data class PaywallData(
     @SerialName("default_locale") val defaultLocale: String,
     @SerialName("localized_strings") internal val localization: Map<String, LocalizedConfiguration>,
 ) {
+
+    /**
+     * @note This allows searching by `Locale` with only language code and missing region (like `en`, `es`, etc).
+     *
+     * @return [LocalizedConfiguration] for the given [Locale], if found.
+     */
+    fun configForLocale(requiredLocale: Locale): LocalizedConfiguration? {
+        return localization[requiredLocale.toString()]
+            ?: localization.takeIf { (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) }?.let { localization ->
```

Comment:
Since paywalls like likely have this or higher minVersion, I think this makes sense 👍

Reply:
oh actually! I thiknk this is not necessary and it's from previous code that was using languageTag, let me check